src/components/_modal/pages/material/Material.page.js

```diff
@@ -159,14 +165,32 @@ const Material = ({ context }) => {
   } = context;
 
   const modal = useModal();
-  const orderMutation = useMutate();
+  const orderAndLoansMutation = useMutate();
+  const [renewed, setRenewed] = useState(false);
+  const [hasError, setHasError] = useState(false);
+  const [renewedDato, setRenewedDato] = useState(null);
+  const { updateUserStatusInfo } = useUser();
+
+  useEffect(() => {
+    //when we show new book, reset variables to avoid showing errors/renwed status and new due date of previous book
+    setRenewed(false);
+    setHasError(false);
+    setRenewedDato(null);
+  }, [title]);
```

Comment:
Kan dette være et id i stedet, så vi ikke får fejl hvis vi har 2 med den samme titel? 

Hvad bruges `materialId` til? Vil den kunne bruges? Alternativt: Ville `useId` kunne bruges, hvis den sendes med til `material`?

Reply:
nå, du har ret. materialId burde være nok, den bruges også som key til de enkelte materialer og for at forny lånet (for fornyelse dog sammen med userId)